hack/build-and-push.sh

```diff
@@ -114,20 +113,18 @@ do
         echo Unknown task in "$task_dir"
         continue
     fi
+
+    # the task files have been copied to the $prepared_task_file location at this point
     repository=${TEST_REPO_NAME:-task-${task_name}}
     tag=${TEST_REPO_NAME:+${task_name}-}${task_version}
-    task_bundle=quay.io/$QUAY_NAMESPACE/${repository}:${tag}
+    task_bundle=quay.io/$QUAY_NAMESPACE/${repository}:${tag}-${task_file_sha}
 
     if digest=$(skopeo inspect --no-tags --format='{{.Digest}}' docker://"${task_bundle}-${task_file_sha}" 2>/dev/null); then
```

Comment:
Does it need to work with `hex-hex`?

Reply:
Oh, I wasn't really clear, I thought it use (i.e. work with) tags as `:0.1-hex-hex`, because on line 120 the `$task_bundle` is `:0.1-hex` and here it would yet again add the `$task_file_sha` to become `:0.1-hex-hex`.

Reply:
Makes sense now. I removed the redundant sha. thanks